src/components/scenes/SettingsScene.tsx

```diff
@@ -313,8 +314,10 @@ export class SettingsSceneComponent extends React.Component<Props, State> {
           <SettingsLabelRow right={autoLogoutRightText} label={lstrings.settings_title_auto_logoff} onPress={this.handleAutoLogout} />
           <SettingsLabelRow right={this.props.defaultFiat.replace('iso:', '')} label={lstrings.settings_title_currency} onPress={this.handleDefaultFiat} />
 
-          <SettingsSwitchRow key="pinRelogin" label={lstrings.settings_title_pin_login} value={this.props.pinLoginEnabled} onPress={this.handlePinToggle} />
-          {this.props.supportsTouchId && (
+          {!isLightAccount ? (
```

Comment:
Convention is to switch this so the null case is first

Reply:
Where do we have such a convention documented? Weird having to think of a flipped condition to visualize what would appear...

Reply:
Having it how I originally had it with null case last also aligns with the logical conditions of nullish coalescing syntax (seen elsewhere in the file).

Reply:
May be documented in past meeting notes. It's fresh in my mind since I've been recently asked to make the same change.